oam/install_scripts/mcs-loadbrm.py.in

```diff
@@ -10,7 +10,7 @@ import glob
 import shutil
 
 API_CONFIG_PATH = '/etc/columnstore/cmapi_server.conf'
-BYPASS_SM_PATH = '/tmp/columnstore_tmp_files/rdwrscratch/BRM_saves'
+BYPASS_SM_PATH = '@ENGINE_LOGDIR@/columnstore_tmp_files/rdwrscratch/BRM_saves'
```

Comment:
How did you test that S3 doesn't store the file?

Reply:
I believe this a temporary directory where nodes are pulling brm files from pm1. It is specifically not stored by SM so that there is not a conflict between nodes trying to write same file. Only used by mcs-loadbrm.py.